src/styles/blocks/stars

```diff
@@ -0,0 +1,22 @@
+.stars {
+  background-color: var(--color-white);
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  margin-right: 4px;
+}
+
+.stars__star:last-of-type {
+  margin-right: 0;
+}
+
+.stars--active > .stars__star:not(:nth-child(n+5)) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
all of those can be nested inside .stars selector - please use `&` [operator ](https://css-tricks.com/the-sass-ampersand/)to access them